src/main/kotlin/org/opensearch/commons/alerting/model/AlertContext.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.commons.alerting.model
+
+/**
+ * This model is a wrapper for [Alert] that should only be used to create a more
+ * informative alert object to enrich mustache template notification messages.
+ */
+data class AlertContext(
```

Comment:
can this class be written in Alerting itself. seems like a construct not used by any other plugin?

Reply:
Good point. I had initially added it in common utils because I was inheriting the Alert data model. I'll move it, and close this PR.